calibration/ADCPAM/CGINS-ADCPAM-48878__20181020.csv

```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+48878,CC_scale_factor1,0.61,DVL Bay 0545; FCP 3304-00150-00111.docs
+48878,CC_scale_factor2,0.61,
+48878,CC_scale_factor3,0.61,
+48878,CC_scale_factor4,0.61,
```

Comment:
Looks good (SNW) – I added DVL Bay S/N and FCP file.

Reply:
Looks good RGT